packages/dicomImageLoader/src/imageLoader/colorSpaceConverters/convertYBRFull422ByPixel.ts

```diff
@@ -6,10 +6,12 @@ export default function (
   useRGBA: boolean
 ): void {
   if (imageFrame === undefined) {
-    throw new Error('decodeRGB: ybrBuffer must not be undefined');
+    throw new Error('convertYBRFull422ByPixel: ybrBuffer must be defined');
   }
   if (imageFrame.length % 2 !== 0) {
```

Comment:
Please add a comment here that the length is divisible by 2 because for 4 pixels (a 2x2 array), there are 4 y and 2 b and 2 r values, so 4+2+2 / 4 = 2 bytes per pixel